src/session.ts

```diff
@@ -375,6 +374,71 @@ export async function getTableHtml(webview: Webview, file: string): Promise<stri
         padding: 0;
         overflow: auto;
     }
+
+    /* Styling for wrapper and header */
+
+    [class*="vscode"] div.ag-root-wrapper {
```

Comment:
Is `[class*="vscode"]` necessary here?

Reply:
This was mostly to have higher priority over the base theme. An alternative would be !important?

Reply:
OK. Makes sense. Maybe a comment on this could be better to clarify the intention.